### ISSUE TITLE
[Bug] Freezes when adding triggers with AppleScript

### ISSUE THREAD

**Issue by JackAppDev** [folivoraAI/BetterTouchTool#2533]
#### Description of bug/feature request/question:
Better touch tool freezes when adding triggers with AppleScript; even the example in the documentation.
```applescript
tell application "BetterTouchTool"
  add_new_trigger "{ \"BTTTriggerClass\" : \"BTTTriggerTypeKeyboardShortcut\", \"BTTPredefinedActionType\" : 5, \"BTTPredefinedActionName\" : \"Mission Control\", \"BTTAdditionalConfiguration\" : \"1179658\", \"BTTTriggerOnDown\" : 1, \"BTTEnabled\" : 1, \"BTTShortcutKeyCode\" : 2, \"BTTShortcutModifierKeys\" : 1179648, \"BTTOrder\" : 3 }"
end tell
```
#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):
All

#### Device information:

Type of Mac: MacBook Pro
macOS version: High Sierra 
BetterTouchTool version: 2.444

**Comment by fifafu** [comment]
Very weird deadlock, but I was able to reproduce it on my other machine.
Will upload a fixed version in a few hours.

**Comment by fifafu** [comment]
It should now be fixed in v2.446 alpha.
